aep/general/0200/aep.md.j2

```diff
@@ -1,5 +1,140 @@
 # Precedent
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/34) for more
-information.
+Many times, APIs are written in ways that do not match new guidance that is
+added to these standards after those APIs have already been released.
+Additionally, sometimes it can make sense to intentionally violate standards
+for particular reasons, such as maintaining consistency with established
+systems, meeting stringent performance requirements, or other practical
+concerns. Finally, as carefully as everyone reviews APIs before they are
+released, sometimes mistakes can slip through.
+
+Since it often is not feasible to fix past mistakes or make the standards serve
+every use case, APIs may be stuck with these exceptions for quite some time.
+Further, since new APIs often base their designs (names, types, structures,
+etc) on existing APIs, it is possible that a standards violation in one API
+could spill over into other APIs, even if original reason for the exception is
+not applicable to the other APIs.
+
+As a result of this problem, it is important to "stop the bleeding" of these
+standards exceptions into new APIs, and additionally document the reasons for
+each exception so that historical wisdom is not lost.
+
+## Guidance
+
+If an API violates the AEP standards for any reason, there **must** be an
+internal comment linking to this document using its descriptive link
+([aep.dev/not-precedent]()) to ensure others do not copy the violations or cite
+the errors as precedent of a "previously approved API".
+
+The comment should also include an explanation of what violates standards and
+why it is necessary. For example:
+
+{% tab proto %}
+
+```proto
+message DailyMaintenanceWindow {
+  // Time within the maintenance window to start the maintenance operations.
+  // It must use the format "HH MM", where HH : [00-23] and MM : [00-59] GMT.
+  // (-- aep.dev/not-precedent: This was designed for consistency with crontab,
+  //     and preceded the AEP standards.
+  //     Ordinarily, this type should be `aep.type.TimeOfDay`. --)
+  string start_time = 2;
+
+  // Output only. Duration of the time window, automatically chosen to be
+  // smallest possible in the given scenario.
+  // (-- aep.dev/not-precedent: This preceded the AEP standards.
+  //     Ordinarily, this type should be `google.protobuf.Duration`. --)
+  string duration = 3;
+}
+```
+{% tab oas %}
+
+**Note:** OAS example not yet written.
+
+{% endtabs %}
+
+**Important:** APIs should only be considered to be precedent-setting if they
+are in beta or GA.
```

Comment:
```suggestion
are in beta or stable.
```

weird nomenclature distinction here: 181 calls the concept of GA stable: https://google.aip.dev/181#stable